src/spyglass/utils/dj_mixin.py

```diff
@@ -119,7 +125,29 @@ def fetch_nwb(self, *attrs, **kwargs):
         AnalysisNwbfile (i.e., "-> (Analysis)Nwbfile" in definition)
         or a _nwb_table attribute. If both are present, the attribute takes
         precedence.
+
+        Additional logic support Export table logging.
         """
+        table, tbl_attr = self._nwb_table_tuple
+        if self.export_id and "analysis" in tbl_attr:
+            logger.debug(
+                f"Export {self.export_id}: fetch_nwb {self.table_name}"
+            )
+            tbl_pk = "analysis_file_name"
+            fname = (self * table).fetch1(tbl_pk)
```

Comment:
`fetch_nwb` usage isn't limited to a single item restriction.  I'm getting errors here when fetching multiple nwb at once